Use CMAKE_INSTALL_LIBDIR instead of hardcoding lib path

Helps build on platforms where libpaths are not lib/ but say lib64/

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -182,7 +182,7 @@ if(NOT FLB_WITHOUT_SHARED_LIB)
     PROPERTIES OUTPUT_NAME fluent-bit)
 
   # Library install routines
-  install(TARGETS fluent-bit-shared LIBRARY DESTINATION lib)
+  install(TARGETS fluent-bit-shared LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
 # Static Library
